app/controllers/api/v0/market_vendors_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
 
   def destroy
     if market_vendor
-      market_vendor.destroy
+      render json: market_vendor.destroy, status: :no_content
     else
```

Comment:
I'm curious as to why we would need render json for this method?

Reply:
I was just styling it, so that the page would look more uniform with a render json: instead of doing it inside the method